core/modules/utils/dom/modal.js

```diff
@@ -122,7 +122,7 @@ Modal.prototype.display = function(title,options) {
 				attributes: {
 					text: {
 						type: "string",
-						value: ( $tw.language == undefined ? "Close" : $tw.language.getString("Buttons/Close/Caption") )
+						value: $tw.language.getString("Buttons/Close/Caption")
```

Comment:
How come `$tw.language == undefined` is not needed when it is in `boot/boot.js` line 139 above? What is the side effect of removing such a check?